src/engraving/CMakeLists.txt

```diff
@@ -37,7 +37,10 @@ set(MODULE_SRC
 
     ${LIBMSCORE_SRC}
 
+    ${CMAKE_CURRENT_LIST_DIR}/draw/color.cpp
+    ${CMAKE_CURRENT_LIST_DIR}/draw/color.h
     ${CMAKE_CURRENT_LIST_DIR}/draw/geometry.h
+    ${CMAKE_CURRENT_LIST_DIR}/draw/globalcolor.h
```

Comment:
I believe that we do not need this enumerator.
At its core, there should be no hardcoded colors in the code.
All colors must be specified in the configuration (some are constant, some can be changed by the user).
The `MScore` class was previously used as a configuration
Now there is an `IEngravingConfiguration` interface for this.
If all colors must be specified in the configuration, then it is logical to specify them in one way.